content/en/developers/prometheus.md

```diff
@@ -17,21 +17,21 @@ aliases:
 
 ## Overview
 
-This page dives into the `PrometheusCheck` interface for more advanced usage, including an example of a simple check that collects timing metrics and status events from [Kube DNS][1]. For information on configuring a basic Prometheus check, see the [Agent Documentation][2].
+This page dives into the `OpenMetricsBaseCheck` interface for more advanced usage, including an example of a simple check that collects timing metrics and status events from [Kube DNS][1]. For information on configuring a basic OpenMetrics check, see the [Agent Documentation][2].
 
-## Advanced usage: Prometheus check interface
+## Advanced usage: OpenMetrics check interface
 
-If you have more advanced needs than the generic check (metrics preprocessing for example) you can write a custom `PrometheusCheck`. It's [the base class][3] of the generic check and it provides a structure and some helpers to collect metrics, events, and service checks exposed via Prometheus. Minimal configuration for checks based on this class include:
+If you have more advanced needs than the generic check (metrics preprocessing for example) you can write a custom `OpenMetricsBaseCheck`. It's [the base class][3] of the generic check and it provides a structure and some helpers to collect metrics, events, and service checks exposed via Prometheus. Minimal configuration for checks based on this class include:
 
 - Overriding `self.NAMESPACE`
 - Overriding `self.metrics_mapper`
 - Implementing the `check()` method
 AND/OR
-- Create method named after the Prometheus metric they will handle (see `self.prometheus_metric_name`)
+- Create method named after the OpenMetric metric they will handle (see `self.prometheus_metric_name`)
 
 ## Writing a custom Prometheus check
 
-This is a simple example of writing a kube DNS check to illustrate the `PrometheusCheck` class usage. The example below replicates the functionality of the following generic Prometheus check:
+This is a simple example of writing a kube DNS check to illustrate the `OpenMetricsBaseCheck` class usage. The example below replicates the functionality of the following generic Prometheus check:
```

Comment:
I think we should change the example to have an actual metric preprocessing function, the current example is not super relevant as it could be done with configuration only. We can probably do this in a follow-up PR

Reply:
Sounds good. I'll create a separate ticket for this